packages/atomic/src/components/search/facets/atomic-popover/atomic-popover.tsx

```diff
@@ -0,0 +1,196 @@
+import {
+  buildSearchStatus,
+  SearchStatus,
+  SearchStatusState,
+} from '@coveo/headless';
+import {Component, h, Listen, State, Element} from '@stencil/core';
+import {
+  BindStateToController,
+  InitializableComponent,
+  InitializeBindings,
+} from '../../../../utils/initialization-utils';
+import {Button} from '../../../common/button';
+import {Hidden} from '../../../common/hidden';
+import {Bindings} from '../../atomic-search-interface/atomic-search-interface';
+import ArrowBottomIcon from 'coveo-styleguide/resources/icons/svg/arrow-bottom-rounded.svg';
+import {
+  InitPopoverEventPayload,
+  ClearPopoversEventPayload,
+} from './popover-event';
+
+/**
+ * @internal
+ * The `atomic-popover` component displays any facet as a popover menu.
+ * @part label-button - The button used to display the popover menu
```

Comment:
I think that your "popover-button" was better 👍 
```suggestion
 * @part popover-button - The button to click to display or hide the popover menu.
```